_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp

```diff
@@ -564,7 +569,7 @@ class GetDefault
             return mfx.TargetKbps * std::max<const mfxU32>(1, mfx.BRCParamMultiplier);
         }
 
-        mfxU32 frN, frD, maxBR = 0xffffffff;
+        mfxU32 frN = 0, frD = 0, maxBR = 0xffffffff;
```

Comment:
Since these are initialized by the std::tie function, I don't think you need to explicitly initialize them, but you can.  I would call this a KW false positive.